tools/eslint-config/package.json

```diff
@@ -26,21 +26,21 @@
     "rules/"
   ],
   "engines": {
-    "node": ">= 8"
+    "node": ">=10"
```

Comment:
If we do this, we need to bump up to `10.x`

Reply:
Yes, I'll change commit message to add as breaking change